src/Mod/Measure/Gui/QuickMeasure.cpp

```diff
@@ -134,7 +135,7 @@ bool QuickMeasure::shouldMeasure(const Gui::SelectionChanges& msg) const
 bool QuickMeasure::isObjAcceptable(App::DocumentObject* obj)
 {
     // only measure shapes
```

Comment:
We should update the comment as well:
```suggestion
    // only measure shapes and datums
```

Or is "datum" already the plural form maybe?

Reply:
_Data_ or _datums_ are permissible plural of _datum_ from a native English speaker

Reply:
ok, but text is 'not datums' actually

Reply:
Ooops, I missed the `!` and thought the opposite was meant